### ISSUE TITLE
A11y_AzureDataStudioSQLMigration_DatabaseMigrationstatus_Migrationinprogress_Keyboard:When keyboard users press enter in the ongoing database under database table header the database window opens in the background and is not visible to the users.

### ISSUE THREAD

**Issue by sspatil123** [microsoft/azuredatastudio#15857]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev4;#SH_AZUREDATASTUDIOSQLMigration_June2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG2.1.1;#AINotInScope;#Keyboard

### Environment Details:
Application Name: SQL Azure Data Studio
Windows Version: Win10

### Repro Steps:

1. Launch the application-Azure Data Studio-Insiders and install extension Azure SQL Migration.
2. Connect to any sql server.
3. Tab till Azure SQL migration Tab and press enter 
4. Tab till "Data migration in progress" and press enter.
5. Tab till ongoing migration database under database table header and press enter.

### Actual:
The database window opens up when opened using mouse but when keyboard users press enter in the ongoing database the window opens in the background and is not visible to the users.

### Expected:
The database window should be visible to the keyboard users.

### User Impact:
Keyboard users will not be able to see the database window.
